jitpack.yml

```diff
@@ -0,0 +1,2 @@
+jdk:
```

Comment:
Do we really need this file?

Reply:
Technically no, but it's handy when someone wants to add flatbuffers git as a gradle dependency.

Reply:
according to the jitpack website, I should be able to do this without such a file. Worse, the only thing this file appears to be doing is specifying a particular kind and version of jdk, which doesn't seem like a good thing to do. Can we remove this file?

Reply:
Without it, jitpack uses jdk 1.6 which is too old to build the code. Sure, we can remove it, but it doesn't hurt to keep it, and we would loose jitpack dependency possibility.

Reply:
Hmpf, yet another top-level file.. ok then.